### ISSUE TITLE
Auto detect low FPS and help the user

### ISSUE THREAD

**Issue by rvanner** [TheGameCreators/GameGuruRepo#2200]
If the frame rate is lower than expected let the user know and suggest they change to a lower setting.

**Comment by UltraVox001** [comment]
Good idea, but a checkbox in the Settings, please.
I don't need Max bugging me with a message every time there's an FPS drop. :)

**Comment by fearlesswee** [comment]
I agree with UltraVox001, this should be completely optional, because I definitely recall this being a sore point in GGC where people felt it was annoying and out of place in standalone games built with GGC.
